modules/blackhole-monitor/src/monitor.py

```diff
@@ -0,0 +1,116 @@
+import boto3
+import logging
+import json
+import os
+
+cloudwatch = boto3.client('cloudwatch')
+ec2 = boto3.client('ec2')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def lambda_handler(event, context):
+  # Get transit gateways
+  get_transit_gateways = ec2.describe_transit_gateways()
+
+  # Loop over transit gateways
+  for transit_gateway in get_transit_gateways['TransitGateways']:
+    logger("Transit Gateway: " + transit_gateway['TransitGatewayId'])
```

Comment:
Should be `logger.info('')`

See https://github.com/ministryofjustice/hmpps-ems-platform-terraform-modules/blob/bb139f18b8a9dd22069a6439fd030014e26ff359/modules/vpn-monitor/src/monitor.py#L43 for ref